conf/ocsci/fdf_pre_ga_deployment.yaml

```diff
@@ -0,0 +1,12 @@
+---
+DEPLOYMENT:
+  fusion_deployment: true
+  fdf_deployment: true
+  channel: v2.0
+  fdf_pre_ga: true
+  fdf_image_tag: v4.18
```

Comment:
Do you have somewhere usage of this file?
Or it's as an example?

If it's bounded to v4.18, I prefer to have also 4.18 in the name of the file, or the plan is to maintain this file and update the version here and keep it as latest?

Reply:
I've created a new directory similar to version specific files we are using for Fusion to maintain these. I'm not sure if we want to keep the config files in `conf/ocsci` for pre-release or not but I can remove these if we don't need them here.